### ISSUE TITLE
Text in SVG with filter is rasterized at low resolution

### ISSUE THREAD

**Issue by sebaseb98** [typst/typst#4926]
### Description

when I want to add a svg, which I created in Inkscape to my document, there occur some errors:

- text gets blurry
- arrows are missing

In the preview everything looks as expected, but when compiling these details somehow get lost

### Reproduction URL

https://typst.app/project/r6AFmbX8lxASSH1fTptN19

### Operating system

macOS

### Typst version

- [X] I am using the latest version of Typst

**Comment by LaurenzV** [comment]
> arrows are missing

This issue is already known and is already fixed in the newest development version, so it will work fine in the next update. See here: https://github.com/typst/typst/issues/3715

> text gets blurry

The reason this is happening is because you have a guassian blur filter that is applied to the text. Since those cannot be represented natively in PDF, Typst instead resorts to rasterizing them, and in this case it unfortunately results in a very low resolution being chosen... Ideally, this should be configurable. With that said, you can avoid this in the first place by not applying a gaussian blur to the text, which you probably wouldn't want to do, anyway. 😄

**Comment by sebaseb98** [comment]
Thank you so much @LaurenzV!

- Arrows are missing ✅
- the blurry text:
For future people finding this: I edited the text of the svg file and removed the filter line in the sections belonging to the text fields. This solved my problem


Is this issue something that will be worked on - I mean technically you closed it by solving my problem but it still would be nice if svgs look the same in the compiled pdf as they look in other viewers.  I leave it up to you to decide :)

**Comment by LaurenzV** [comment]
Yes, let’s keep it up until there is a way to adjust that from Typst.

**Comment by laurmaedje** [comment]
> Yes, let’s keep it up until there is a way to adjust that from Typst.

Do we really want to make that adjustable? That seems like a tad low-level for me. Can't we just increase the resolution to something that works well?

**Comment by LaurenzV** [comment]
> Do we really want to make that adjustable? That seems like a tad low-level for me. Can't we just increase the resolution to something that works well?

I guess that would work too, but we would have to adapt it depending on how big it ends up in the final PDF.

**Comment by EpicEricEE** [comment]
Maybe this can somehow be combined with #2554 to a single parameter for image resolutions, which for SVG then only accounts for the rasterized part?

**Comment by nineff** [comment]
The current rasterization resolution also leads to blurry edges at objects which should otherwise have a clearly defined corner. See the attached files.[svg.pdf](https://github.com/user-attachments/files/17316801/svg.pdf)
![test](https://github.com/user-attachments/assets/76eee9d0-26c6-414a-8e2e-188f9adbb2cd)
